OpenRA.Game/Network/UnitOrders.cs

```diff
@@ -86,7 +86,7 @@ internal static void ProcessOrder(OrderManager orderManager, World world, int cl
 							if (orderManager.LocalClient != null && client != orderManager.LocalClient && client.Team > 0 && client.Team == orderManager.LocalClient.Team)
 								suffix += " (Ally)";
 
-							TextNotificationsManager.AddChatLine(client.Name + suffix, message, client.Color);
+							TextNotificationsManager.AddChatLine(client.Name + suffix, clientId, message, client.Color);
```

Comment:
Trivial. But if clientId is the "key", i'd use it as a first argument. It is not they key, but more unique than the prefix.

Optionally add function `AddSystemMessage`  where `clientId` can be left out indicating system/-1.

Also for clarity you could make `-1` a const value i.e. `ClientIdSystem`.

Callers might as well pass a single TextNotifcation arguments if there are always goign to be 4+ arguments.

Reply:
Moved `clientId` as the first argument and added `ClientIdSystem` constant